rpm/buildah.spec

```diff
@@ -69,6 +72,9 @@ BuildRequires: libseccomp-devel
 %endif
 Requires: libseccomp >= 2.4.1-0
 Suggests: cpp
+%if %{defined sequoia}
+Requires: podman-sequoia
+%endif
```

Comment:
Remind me again why we have a packaging dependency on podman, when podman has a library dependency on buildah?